src/js/site.js

```diff
@@ -0,0 +1,76 @@
+window.addEventListener("DOMContentLoaded", () => {
```

Comment:
@will0101 Notes on what I changed in the JS

- Moved it to a separate file, still including it in the DOM in a `script` element. It's small enough that's still fine
- Updated the way we add the form submit handler so this works for multiple forms on a single page
- Added "Form" to the `showSucces`, `showError`, `toggleBtn` function names so they're a bit more descriptive
- Also on those, changed them to take parameters instead of relying on outside variables. This makes them side-effect free. They only operate on what is in scrope